src/utilities/include/public.h

```diff
@@ -19,17 +19,14 @@
 #include <glm/gtc/constants.hpp>
 #include <glm/gtx/compatibility.hpp>
 #include <glm/gtx/rotate_vector.hpp>
+#include <iomanip>
+#include <iostream>
 #include <limits>
 #include <memory>
+#include <sstream>
 #include <unordered_map>
 #include <vector>
 
-#ifdef MANIFOLD_DEBUG
-#include <iomanip>
-#include <iostream>
-#include <sstream>
-#endif
```

Comment:
Why this change? Part of my purpose here was to keep the WASM build size down since including things like `std::string` are notoriously large.

Reply:
Oh I forgot to revert this change, I was trying to debug the ubuntu failure when using system provided gtest.